cfgov/unprocessed/apps/regulations3k/css/reg-navigation.less

```diff
@@ -3,65 +3,128 @@
    Custom eRegs secondary navigation styles
    ========================================================================== */
 
-.o-secondary-navigation {
-    &__no-children {
-        .respond-to-max( @bp-sm-max, {
+// Regs3K sidebar
+.o-regs3k-sidebar {
+    .respond-to-max( @bp-sm-max {
+        background-color: @gray-5;
+        border-bottom: 1px solid @gray-40;
+        border-top: 1px solid @gray-40;
+        margin-bottom: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
+    } );
+}
+
+// Regs3K secondary nav sections
+.o-regs3k-sections {
+
+    .o-expandable {
+        margin-bottom: 0;
```

Comment:
This also _shouldn't_ be necessary but I'm not 100% certain, maybe we can pair on a call about it if you're finding it is.